### DISCUSSION TITLE
How is the UI option of filesize implemented in v9?

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#4690] [Archive]
![image](https://user-images.githubusercontent.com/8030134/112566642-5ae23480-8e1a-11eb-8b88-b27517a28e0a.png)

**Comment by rijkvanzanten** [comment]
I'm not sure I know what you're talking about 🤔

**Reply by cdwmhcc** [reply]
In v9, I set the file size in the field settings, but it is not displayed like the red box in the picture above.

It is shown in the following picture in v9. I hope it has KB/MB/GB options like v8.

![image](https://user-images.githubusercontent.com/8030134/112642994-d675cd00-8e7e-11eb-9e46-6677831da8b6.png)

![image](https://user-images.githubusercontent.com/8030134/112642688-85fe6f80-8e7e-11eb-9bec-8afde735266c.png)

**Reply by rijkvanzanten** [reply]
Ohh I see. There's no `filesize` _interface_ in v9 as of right now, only a _display_

**Reply by cdwmhcc** [reply]
Okay, I'll use it like this for now.The conversion is a bit troublesome.😁😁
